src/tools/wasm-reduce.cpp

```diff
@@ -505,6 +505,7 @@ struct Reducer : public WalkerPass<PostWalker<Reducer, UnifiedExpressionVisitor<
               case i64: fixed = builder->makeUnary(WrapInt64, child); break;
               case f32: fixed = builder->makeUnary(TruncSFloat32ToInt32, child); break;
               case f64: fixed = builder->makeUnary(TruncSFloat64ToInt32, child); break;
+              case v128: assert(false && "v128 not implemented yet");
```

Comment:
instead of crashing, let's make the reducer ignore such things, and add a TODO of that assertion text.

To skip it here, I think just `continue` will work.